rfcs/src/template.md

```diff
@@ -0,0 +1,58 @@
+- Feature Name: *Fill me with pretty name and a unique ident. E.g: New Feature (`new_feature`)*
+- Feature Request Issue: *Link to issue*
+- RFC PR: *Link to original PR*
+- Status: *One of the following: [Under Review | Unstable | Stable | Cancelled]*
```

Comment:
Per @danielsn suggestion, we should add an RFC version here and an optional link to a proof-of-concept.

I believe the RFC version should be bumped with subsequent PRs that update the RFC not during the PR feedback.